Dockerfile

```diff
@@ -20,8 +20,7 @@ RUN apk add --no-cache -U \
 
 RUN pip install mcstatus yq
 
-ARG HEALTHCHECK_DEFAULT_START=5m
```

Comment:
I liked the build arg, I was just wanting a value of `1m` instead of `5m`

Reply:
I liked it too but had a hell of a type trying to get it to work so I just revered to a know good state.

Reply:
Oh, I wonder if build args don't carry over to the health check's execution environment.